tests/integration-test/src/tests/basic.rs

```diff
@@ -296,7 +296,11 @@ fn test_load_unload_kretprobe() {
 
 #[integration_test]
 fn test_pull_bytecode() {
-    std::fs::remove_dir_all(BYTECODE_IMAGE_CONTENT_STORE).unwrap();
+    let path = PathBuf::from(BYTECODE_IMAGE_CONTENT_STORE);
```

Comment:
I hit that one too, and fixed it like this in my last pr:
    let _ = std::fs::remove_dir_all(BYTECODE_IMAGE_CONTENT_STORE);

Reply:
Hrm I don't want to remove the error though... i.e we should know it failed unless it was because the dir already existed

Reply:
True.  Your solution will cover the case where the dir exists, but the delete fails anyway.

Reply:
I'll just drop that change from my pr.  I hit the issue because I ran `sudo ./scripts/setup.sh uninstall`, which deletes that dir, before running int test.